src/parse.c

```diff
@@ -1608,6 +1618,21 @@ handle_routes(NetplanParser* npp, yaml_node_t* node, const void* _, GError** err
         if (!process_mapping(npp, entry, routes_handlers, NULL, error))
             goto err;
 
+        /* Set the default scope, according to type */
+        if (!route->scope) {
+            if (   g_ascii_strcasecmp(route->type, "local") == 0
+                || g_ascii_strcasecmp(route->type, "nat") == 0)
+                route->scope = (g_strdup("host"));
+            /* Non-gatewayed unicast routes are scope:link, too */
```

Comment:
Should we rather print a warning, suggesting to add `scope: link` in this case, instead of doing it implicitly?

Reply:
I'd rather not. The default behavior seems logical to me and matches what `ip route` does by default. I'd rather have `netplan get` grow the capacity to display the full configuration (which should be easy after #252), which could help whoever forgot their gateway.